src/bundle/Resources/config/default_settings.yaml

```diff
@@ -1,5 +1,6 @@
 parameters:
   ibexa.graphql.schema.should.extend.ezurl: false
+  ibexa.graphql.schema.ezobjectrelationlist.enable_pagination: false
```

Comment:
If this goes for 4.6 (not saying it should), maybe we could think about some forward compability and use 
`ibexa.graphql.schema.ibexa_object_relation_list.enable_pagination` as default and `ibexa.graphql.schema.ezobjectrelationlist.enable_pagination` as deprecated fallback?